### PR TITLE
Command args

### PR DESCRIPTION
**Pull request by pierlauro**
Ability to specify in the singularity-compose the arguments to pass to an instance's startscript

### CODE REVIEW

**Comment by pierlauro** [comment]
I decided to use the `command` directive to be aligned with the docker-compose syntax

File: docs/spec/spec-1.0.md [hunk 1]
```diff
@@ -48,6 +48,21 @@ Since Singularity does not (currently) have control over custom networking,
 all instance ports are mapped to the host (localhost) and we don't have any
 configuration settings to control this (how to handle ports?)
 
+## Startscript arguments
+
+It is possible to use the `command` option to pass arguments to an instance's
+startscript.
+
+The following example shows how to pass the arguments `arg0 arg1arg2` to the
```
Comment (vsoch) [inline docs/spec/spec-1.0.md]: `arg1 arg2 arg3` (just missing a space between arg 1 and 2!)

File: scompose/project/instance.py [hunk 2]
```diff
@@ -139,6 +140,10 @@ def set_ports(self, params):
 
     # Commands
 
+    def set_args(self, params):
```
Comment (vsoch) [inline scompose/project/instance.py]: Perfect.

File: docs/spec/spec-1.0.md (hunk 1 above)
Comment (vsoch) [reply docs/spec/spec-1.0.md]: No worries about this, I'll fix it!

**Comment by vsoch** [comment]
okay, I just did one more push because we needed to add a line to circleci/config.yml to make sure that the test runs in circle, and also to fix the tiny typo in the docs. As soon as this passes we should be good to go.

**Comment by vsoch** [comment]
Woot! Here is the new test:

Screenshot_2020-03-18 run-scompose-ci-tests (61) - singularityhub singularity-compose

**Comment by vsoch** [comment]
https://pypi.org/project/singularity-compose/0.0.19/

**Comment by pierlauro** [comment]
Perfect, thank you for the fast review!

**Pull request merged**